x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/deferred_assets_warning.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiIcon, EuiToolTip } from '@elastic/eui';
+
+export const DEFERRED_ASSETS_WARNING_LABEL = i18n.translate(
+  'xpack.fleet.packageCard.reauthorizationRequiredLabel',
+  {
+    defaultMessage: 'Reauthorization required',
+  }
+);
+
+export const DEFERRED_ASSETS_WARNING_MSG = i18n.translate(
+  'xpack.fleet.epm.packageDetails.assets.deferredInstallationsMsg',
+  {
+    defaultMessage:
+      'This package has at least one deferred installation which requires additional permissions to install and operate correctly.',
```

Comment:
@qn895 @szabosteve should we adapt all this text if logging back in as a user who does have the required permissions to run the transform? This can give the impression that the current user doesn't have the correct permissions when in fact they do:

<img width="905" alt="image" src="https://user-images.githubusercontent.com/7405507/233062924-93298d6f-aa00-4812-adeb-afc2dc07c61a.png">

Reply:
If we have the same option as above to display 2 different messages depending on the permissions of the logged-in user, then we can use the current text for the scenario where the user doesn't have the required privileges. We can display the following for users with the right privileges: `This package has one deferred installation. Complete the installation to operate the package correctly.`

Reply:
Updated here 
<img width="1493" alt="Screen Shot 2023-04-19 at 14 47 02" src="https://user-images.githubusercontent.com/43350163/233191656-99650d44-54de-47bb-a317-aae45eb99166.png">
<img width="1491" alt="Screen Shot 2023-04-19 at 14 47 35" src="https://user-images.githubusercontent.com/43350163/233191661-422c3e43-5a60-4d1f-830a-1b52098a306c.png">